lib/ft817/ft817.cpp

```diff
@@ -264,7 +266,7 @@ bool FT817::getVFO()
 {
 	MSB = 0x00;	// set the address to read
 	LSB = 0x55;
-	return (bool)(readEEPROM() & 0b00000001);
+	return (bool((readEEPROM() & 0b00000001)));    // debug in progress here... g7uhn

```

Comment:
This is where I've got to at this time (19:00 UTC Sunday) and will focus on next.

Reply:
As per C standard is no matter if you say byte(a) or (byte)a, but if a if a compound calculation then it must have () around it